### ISSUE TITLE
Bug: `URL.createObjectURL` incorrectly marked as experimental

### ISSUE THREAD

**Issue by xatlasm** [eslint-community/eslint-plugin-n#470]
### Environment

Node version: v22.17.1
npm version: 10.8.2  
ESLint version: v9.32.0
eslint-plugin-n version: 17.21.3
Operating System: macOS 15.6 (Darwin 24.6.0) on Apple Silicon (arm64)

### What rule do you want to report?

n/no-unsupported-features/node-builtins

### Link to Minimal Reproducible Example

https://stackblitz.com/edit/stackblitz-starters-ggce9r4h

### What did you expect to happen?

`URL.createObjectURL` should not trigger an error when using Node.js 22+, since this API has been stable (not experimental) since Node.js 22.17.0.

According to the [MDN documentation](https://developer.mozilla.org/en-US/docs/Web/API/URL/createObjectURL_static), `URL.createObjectURL` has "Full support" in Node.js since version 16.7. It was only marked as stable as of 22.17.0, but it had not changed since 16.7.

### Participation

- [x] I am willing to submit a pull request for this issue.

### Additional comments

Fix: [lib/unsupported-features/node-builtins-modules/url.js](https://github.com/eslint-community/eslint-plugin-n/blob/91a56d01322d1f3da8571b5358fd641a15a1e42a/lib/unsupported-features/node-builtins-modules/url.js#L15)

```js
URL: {
    [READ]: { supported: ["7.0.0", "6.13.0"] },
    canParse: { [READ]: { supported: ["19.9.0"] } },
    createObjectURL: { [READ]: { supported: ["16.7.0"] } },
    revokeObjectURL: { [READ]: { supported: ["16.7.0"] } },
},
```

**Comment by scagood** [comment]
This should be done as part of the node version updates (https://github.com/eslint-community/eslint-plugin-n/issues/450).

---

That being said, this does mean the correct shape should now be:

```ts
URL: {
    [READ]: { supported: ["7.0.0", "6.13.0"] },
    canParse: { [READ]: { supported: ["19.9.0"] } },
    createObjectURL: { [READ]: { experimental: ["16.7.0"], supported: ["22.17.0"] } },
    revokeObjectURL: { [READ]: { experimental: ["16.7.0"], supported: ["22.17.0"] } },
},
```
